src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
It is good practice to use separate variables to store delimiters. This makes the code more understandable and easier to debug.
Example: 
```suggestion

  const oldSeparator = fromFormat.pop();
  const newSeparator = toFormat.pop(); 

  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
```